dictionaries/CallMap.php

```diff
@@ -11121,7 +11121,7 @@
 'session_set_save_handler' => ['bool', 'open'=>'callable(string,string):bool', 'close'=>'callable():bool', 'read'=>'callable(string):string', 'write'=>'callable(string,string):bool', 'destroy'=>'callable(string):bool', 'gc'=>'callable(string):bool', 'create_sid='=>'callable():string', 'validate_sid='=>'callable(string):bool', 'update_timestamp='=>'callable(string):bool'],
 'session_set_save_handler\'1' => ['bool', 'open'=>'SessionHandlerInterface', 'close='=>'bool'],
 'session_start' => ['bool', 'options='=>'array'],
-'session_status' => ['int'],
+'session_status' => ['PHP_SESSION_ACTIVE|PHP_SESSION_NONE|PHP_SESSION_ACTIVE'],
```

Comment:
```suggestion
'session_status' => ['1|2'],
```

Reply:
Also you had a duplicate there

Reply:
I took a different approach. Hope it works.

Reply:
That's clever, but I would prefer values though. Many of the constants in PHP are not really constants - e.g. `PHP_BINARY`, `PHP_INT_SIZE`, `PASSWORD_DEFAULT`, etc. Using constants in the callmap the way you did establishes an unsafe example for future contributors / maintainers who may not be necessarily aware of those details.